src/Input/index.js

```diff
@@ -1,7 +1,9 @@
 import React from "react";
 
-const Input = () => {
-  return <input />;
+
+const Input = (props) => {
+  
+  return <input value={props.value} onChange={props.onChange} />;
```

Comment:
```suggestion
const Input = ({ value, onChange, ...props}) => {
  
  return <input value={value} onChange={onChange} {...props} />;
```
또는
```suggestion
const Input = (props) => {
  
  return <input {...props} />;
```
nit: 이렇게도 가능하다는 사실

공부할 부분: 왜 다른 {...props}는 어떤 역할을 할까?